solidity/test/isms/MultisigIsm.t.sol

```diff
@@ -187,7 +187,8 @@ abstract contract AbstractMultisigIsmTest is Test {
         // changing single byte in metadata should fail signature verification
         uint256 index = uint256(seed) % metadata.length;
         metadata[index] = ~metadata[index];
-        assertFalse(ism.verify(metadata, message));
+        vm.expectRevert();
```

Comment:
Is this intended? I dont see why this is now expected to revert when returning false previously?

Reply:
Actually generally seems like an unrelated change?